basic-features.md

```diff
@@ -1,80 +1,136 @@
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                                      | Deprecated   | Deprecated   | Deprecated   | Y            | Y            | Y            |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                    | Y            | Y            | Y            | N            | N            | N            |
+| **Management, observability and tools**                                                                  | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [TiDB Dashboard](/dashboard/dashboard-intro.md)                                                          | Y            | Y            | Y            | N            | N            | N            |
+| [SQL diagnostics](/information-schema/information-schema-sql-diagnostics.md)                             | Experimental | Experimental | Experimental | N            | N            | N            |
+| [Information schema](/information-schema/information-schema.md)                                          | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Metrics schema](/metrics-schema.md)                                                                     | Y            | Y            | Y            | N            | N            | N            |
+| [Statements summary tables](/statement-summary-tables.md)                                                | Y            | Y            | Y            | N            | N            | N            |
+| [Slow query log](/identify-slow-queries.md)                                                              | Y            | Y            | Y            | Y            | Y            | Y            |
+| [TiUP deployment](/tiup/tiup-overview.md)                                                                | Y            | Y            | Y            | N            | N            | N            |
+| Ansible deployment                                                                                       | N            | N            | Deprecated   | Y            | Y            | Y            |
+| [Kubernetes operator](https://docs.pingcap.com/tidb-in-kubernetes/)                                      | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Built-in physical backup](/br/backup-and-restore-use-cases.md)                                          | Y            | Y            | Y            | N            | N            | N            |
+
+[^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details. [^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
```

Comment:
How about adding a blank line to separate "[^1]" and "[^2]"?

Reply:
I had it with a separate line, but there is a linter check for hard breaks.

Reply:
```suggestion
[^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.

[^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
```